tools/vscode-debugging/conf/launch.json

```diff
@@ -0,0 +1,49 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "interactive",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "preLaunchTask": "pre-debug",
+            "program": "${workspaceFolder}/cmd/launcher",
+            "args": [
+                "interactive",
+                "--osquery_flag=allow-unsafe",
+                "--osquery_flag=verbose",
+                // add atc config to the path below to load atc config
+                "--osquery_flag=config_path=${workspaceFolder}/debug/kolide-atc-config.json",
```

Comment:
Do you think we should include this? I might just omit it, and advocate we fold it into `interactive`

Reply:
removed